tests/test_runner.py

```diff
@@ -81,6 +81,20 @@ def test_runner_run_ok(self):
         nasl_file = _here / "plugins" / "test.nasl"
         content = nasl_file.read_text(encoding="latin1")
 
+        # Check sys exit 1
+        included_plugins = [
+            "CheckMissingDescExit",
+        ]
+        runner = Runner(
+            n_jobs=1,
+            included_plugins=included_plugins,
+            term=self._term,
+        )
+        with redirect_stdout(io.StringIO()) as _:
```

Comment:
Sorry i didn't see that initially. Should be just
```python
with redirect_stdout(io.StringIO()):
```

Reply:
removed with [8f84c32](https://github.com/greenbone/troubadix/commit/8f84c324bd95ff6029f29258004e44dc9d6ae657)